addons/project/static/src/js/tours/project.js

```diff
@@ -66,67 +66,181 @@ registry.category("web_tour.tours").add('project_tour', {
     width: 350,
     position: "bottom"
 }, {
-    trigger: ".o-mail-Chatter-topbar button:contains(Activities)",
+    trigger: ".o-mail-Chatter-topbar button.o-mail-Chatter-activity",
     extra_trigger: '.o_form_project_tasks',
     content: markup(_t("Create <b>activities</b> to set yourself to-dos or to schedule meetings.")),
     position: "bottom",
 }, {
-    trigger: ".modal-dialog .btn-primary",
+    trigger: ".modal-dialog button[name='action_schedule_activities']",
     extra_trigger: '.o_form_project_tasks',
     content: _t("Schedule your activity once it is ready."),
     position: "bottom",
     run: "click",
 }, {
     trigger: ".o_field_widget[name='user_ids']",
     extra_trigger: '.o_form_project_tasks',
-    content: _t("Assign a responsible to your task"),
+    content: _t("Assign a responsible to your task."),
     position: "right",
     run() {
         document.querySelector('.o_field_widget[name="user_ids"] input').click();
     }
 }, {
     trigger: ".ui-autocomplete > li > a:not(:has(i.fa))",
     auto: true,
+}, {
+    trigger: ".o_field_project_task_state_selection .dropdown-toggle",
+    content: markup(_t("You can also change the <b>state</b> of the task here.")),
+    position: "bottom",
+    run: "click",
+}, {
+    trigger: ".o_field_project_task_state_selection .dropdown-toggle",
+    content: markup(_t("Click again to close the dropdown.")),
+    position: "left",
+    run: "click",
+}, {
+    trigger: 'a[name="sub_tasks_page"]',
+    content: _t('Open sub-tasks notebook section.'),
+    run: 'click',
+}, {
+    trigger: '.o_field_subtasks_one2many .o_list_renderer a[role="button"]',
+    content: _t('Add a sub-task.'),
+    run: 'click',
+}, {
+    trigger: '.o_field_subtasks_one2many div[name="name"] input',
+    content: markup(_t('Give the sub-task a <b>name</b>.')),
+    run: 'text New Sub-task'
+}, {
+    trigger: ".breadcrumb-item:not(.active):last",
+    extra_trigger: '.o_form_project_tasks',
+    content: markup(_t("Let's go back to the <b>kanban view</b> to have an overview of your next tasks.")),
+    position: "right",
+    mobile: false,
+    run: 'click',
+}, {
+    trigger: ".o_back_button",
+    extra_trigger: '.o_form_project_tasks',
+    content: markup(_t("Let's go back to the <b>kanban view</b> to have an overview of your next tasks.")),
+    position: "right",
+    mobile: true,
+    run: 'click',
+}, {
+    trigger: '.o_kanban_record .o_field_project_task_state_selection button',
+    content: markup(_t("Check the <b>state</b> of the task here again.")),
+    position: "bottom",
+}, {
+    trigger: '.o_kanban_record .o_field_project_task_state_selection .dropdown-menu .dropdown-item:has(.fa-times-circle)',
+    content: markup(_t("You can use the last two <b>states</b> proposed <i>Canceled</i> and <i>Done</i> to <b>close</b> the task, \
+    closed task won't appear in the kanban view by default, try to set this task as <i>Canceled</i>.")),
```

Comment:
```suggestion
    content: markup(_t("You can use the last two <b>states</b> (<i>Cancelled</i> and <i>Done</i>) to <b>close</b> the task. \
    Closed task won't appear in the kanban view by default. Try to set this task as <i>Cancelled</i>.")),
```
Also, I would either not capitalize the stage names here or also do it in other steps

Reply:
Capitalized everywhere, like in the dropdown choice